specifications/cdsc-wg1-02.md

```diff
@@ -1 +1,799 @@
+* `pushed_authorization_request_endpoint` - _URL_ - (REQUIRED) OAuth's [Pushed Authorization Requests](https://www.rfc-editor.org/rfc/rfc9126) functionality is required.
+
+In addition to the above additionally required set of OAuth [Authorization Server Metadata](https://www.rfc-editor.org/rfc/rfc8414#section-2) values, this specification clarifies use of the following OAuth sandard values:
+
+* `response_types_supported` - _Array[string]_ - (REQUIRED) The response types in this array MUST represent a union of all `response_types_supported` values contained in the `cds_scope_descriptions` object.
+* `grant_types_supported` - _Array[string]_ - (REQUIRED) The response types in this array MUST represent a union of all `grant_types_supported` values contained in the `cds_scope_descriptions` object.
+* `token_endpoint_auth_methods_supported` - _Array[string]_ - (REQUIRED) The response types in this array MUST represent a union of all `token_endpoint_auth_methods_supported` values contained in the `cds_scope_descriptions` object.
+
+In addition to OAuth capabilities included in the metadata object, this specification adds the following Carbon Data Specification (CDS) values:
+
+* `cds_oauth_version` - _string_ - (REQUIRED) The version of the CDS-WG1-02 Client Registration specification that the Server has implemented, which for this version of the specification is `v1`
+* `cds_human_registration` - _URL_ - (REQUIRED) Where Clients who do not have the technical capacity to use the `registration_endpoint` can visit to manually register a Client using a web browser
+* `cds_human_directory` - _URL_ - (REQUIRED) A public web interface where users may browse the list of Clients who have registered and set their Client Settings `profile_visibility` to `listed`
+* `cds_test_accounts` - _URL_ - (REQUIRED) Where Clients can find developer documentation on what test account credentials may be used for testing OAuth `response_type=code` authorization requests
+* `cds_clients_api` - _URL_ - (REQUIRED) The base url for the [Clients API](#clients-api)
+* `cds_client_settings_api` - _URL_ - (REQUIRED) The base url for the [Client Settings API](#client-settings-api)
+* `cds_client_updates_api` - _URL_ - (REQUIRED) The base url for the [Client Updates API](#client-updates-api)
+* `cds_scope_credentials_api` - _URL_ - (REQUIRED) The base url for the [Scope Credentials API](#scope-creds-api)
+* `cds_grants_api` - _URL_ - (REQUIRED) The base url for the [Grants API](#grants-api)
+* `cds_directory_api` - _URL_ - (REQUIRED) The base url for the [Directory API](#directory-api)
+* `cds_scope_descriptions` - _Map[[ScopeDescription](#scope-descriptions-format)]_ - (REQUIRED) An object providing additional information about what Scope values listed in `scopes_supported` mean. This object MUST include a key for each Scope listed in `scopes_supported` with a [Scope Description](#scope-descriptions-format) as that key's value.
```

Comment:
Out of curiosity, does this need to be a map and not an array?
The `id` of the the Scope Description object is being used as the key in this map. Instead of duplicating, can we just avoid adding another key here and just use the `id` from the scope object when it needs to be referenced? Is it to optimize lookups?